tests/snuba/api/endpoints/test_organization_events.py

```diff
@@ -123,3 +126,71 @@ def test_project_filtering(self):
                 event_2.event_id,
             ]
         )
+
+    def test_stats_period(self):
+        self.login_as(user=self.user)
+
+        project = self.create_project()
+        project2 = self.create_project()
+        group = self.create_group(project=project)
+        group2 = self.create_group(project=project2)
+        event_1 = self.create_event('a' * 32, group=group, datetime=self.min_ago)
+        self.create_event('b' * 32, group=group2, datetime=self.day_ago)
+
+        url = reverse(
+            'sentry-api-0-organization-events',
+            kwargs={
+                'organization_slug': project.organization.slug,
+            }
+        )
+        url = '%s?statsPeriod=2h' % (url,)
+        response = self.client.get(url, format='json')
+
+        assert response.status_code == 200, response.content
+        assert len(response.data) == 1
+        assert sorted(map(lambda x: x['eventID'], response.data)) == sorted(
+            [
+                event_1.event_id,
+            ]
+        )
```

Comment:
You do this a couple times in the test, is it worth having an assertion method on the test class?